src/CMakeLists.txt

```diff
@@ -337,7 +337,18 @@ if(USE_LENSFUN)
   endif()
 endif()
 
-foreach(lib ${OUR_LIBS} GIO GThread GModule PangoCairo Rsvg2 LibXml2 Sqlite3 CURL PNG JPEG TIFF LCMS2 JsonGlib)
+find_package(Sqlite3 3.15 REQUIRED)
+if(NOT Sqlite3_VERSION VERSION_LESS 3.24)
+  add_definitions("-DHAVE_SQLITE_324_OR_NEWER")
+  message(STATUS "SQLITE version greater or newer than 3.24")
+else()
+  message(STATUS "SQLITE version less than 3.24, some features may be downgraded")
```

Comment:
Similarly here you should use `Sqlite3` instead of `SQLITE`. And maybe instead of "less than" I'd use "lower than" or "earlier than"?

Reply:
True for both. I'll fix that. Thanks